lib/manager/npm/post-update/pnpm.ts

```diff
@@ -72,7 +71,7 @@ export async function generateLockFile(
     }
 
     await exec(`${cmd} ${args}`, execOptions);
-    lockFile = await readFile(lockFileName, 'utf8');
+    lockFile = await readLocalFile(lockFileName, 'utf8');
```

Comment:
Same issue as before. Is cwd the path to package file?

Reply:
I tried to fix it. I think cwd is the absolute path to the "current working dir", meaning the directory where package lock is contained.